examples/parametric/compare_sfhs.py

```diff
@@ -0,0 +1,55 @@
+import matplotlib.pyplot as plt
```

Comment:
In order for an example to show a plot in the examples gallery it needs to follow a specific naming and formatting convention. See the [contributing guidelines](https://github.com/flaresimulations/synthesizer/blob/main/docs/CONTRIBUTING.md#adding-example-scripts) for details.